assets/assets.go

```diff
@@ -22,7 +22,7 @@ var (
 // TimeService ...
 //go:generate go run github.com/golang/mock/mockgen -destination mocks/time_service_mock.go -package mocks code.vegaprotocol.io/vega/assets TimeService
 type TimeService interface {
```

Comment:
No, it is not: https://github.com/vegaprotocol/vega/pull/5475/commits/810006e924b64d1dde48ec5fc8e5322173bf3563
Also, it seems to me that for most engines will be better to directly import vegatime/mocks, TimeService behaviour, etc, 
because most do not implement it at all, but only used for mocks generation.